DSCResources/MSFT_xSqlServerInstall/MSFT_xSqlServerInstall.psm1

```diff
@@ -69,6 +77,8 @@ function Set-TargetResource
         [string] $SourcePath,
 
         [PSCredential] $SourcePathCredential,
+        
+        [string]$VersionID="120",
```

Comment:
@TravisEz13 Can you please explain your comment further? the $VersionID parameter is only defined once in each of the "Test-", "Set-", and "Get-" functions. It has a default value assigned so that it will work with the latest release by default. I am not sure why you are requesting that it be defined in a common location.